core/jobs/job_options.py

```diff
@@ -63,6 +63,8 @@ def __init__(
             region=feconf.GOOGLE_APP_ENGINE_REGION,
             temp_location=feconf.DATAFLOW_TEMP_LOCATION,
             staging_location=feconf.DATAFLOW_STAGING_LOCATION,
+            experiments=['use_runner_v2', 'enable_recommendations'],
+            extra_packages=['./build/oppia-beam-job-0.0.1.tar.gz'],
```

Comment:
I think this string needs to be pulled out to a config file somewhere? Shouldn't have it buried deep within the code if it needs to track an external repo.

(Should also have backend test validating that this string is in sync with other things, e.g. wherever we specify which version of oppia-beam-job we need to download.)

Reply:
Done.

Reply:
We don't version this and I don't think it is needed.

Reply:
Hm, perhaps explain the reasoning for not versioning in the code.

Reply:
Done.